### PR TITLE
 Fix #21 #22

### LINKED ISSUE #21: GPServiceAccount should be able to initialize without serviceInstanceName

### PR DESCRIPTION
**Pull request by sid41**
Fix #21 #22

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -17,8 +17,7 @@ What is this?
 This is a demo Python app which uses the Globalization Pipeline service on
 Bluemix. Note that this app is designed to be run on Bluemix.
 
-A running instance of the demo can be found [here](
-http://gp-python-client-demo.mybluemix.net/).
+A running instance of the demo can be found [here](https://gp-py-client.mybluemix.net/).
```
Comment (srl295) [inline demo/README.md:20]: is there a 'new' style domain available? instead of mybluemix.net 
May be able to just add a route to the current app.

**Review by srl295 — APPROVED** [review]
lgtm with a question

**Review by jugu — APPROVED** [review]
lgtm

File: demo/README.md (hunk 1 above)
Comment (sid4) [reply demo/README.md:20]: I tried using g11n-...cloud.ibm.com domain, had dns issues so just used mybluemix.net.

Comment (srl295) [reply demo/README.md:20]: ^ let's move forward then.

**Pull request merged**